frontend/public/components/utils/list-dropdown.jsx

```diff
@@ -45,7 +45,10 @@ class ListDropdown_ extends React.Component {
       return;
     }
 
-    this.setState(({ selectedKey }) => {
+    this.setState(() => {
+      const selectedKey = _.isEmpty(nextProps.selectedKey)
+        ? null
+        : `${nextProps.selectedKey}-${nextProps.selectedKeyKind}`;
```

Comment:
Don't we need to check if `nextProps.selectedKeyKind` is set first (as we do in the constructor)?